v2/api/managedidentity/v1api20220131preview/structure.txt

```diff
@@ -9,7 +9,8 @@ FederatedIdentityCredential: Resource
 ├── Spec: Object (8 properties)
 │   ├── Audiences: string[]
 │   ├── AzureName: string
-│   ├── Issuer: *string
+│   ├── Issuer: Validated<*string> (1 rule)
+│   │   └── Rule 0: Pattern: "^[a-zA-Z][a-zA-Z0-9+-.]*:[^\\s]*$"
```

Comment:
This change is technically breaking, as we could have accepted a resource previously that is no longer valid. We should maybe tag this as breaking and note it in the next release?